Ash.FrontEnd/src/global/style.less

```diff
@@ -0,0 +1,20 @@
+.ash-section {
+  margin: 10px;
+  border-radius: 10px;
+  background-color: #333;
+
+  div {
+    padding: 20px 20px 10px;

```

Comment:
Imo, 3-arg padding/margin rules are confusing cause I can never remember which rule is copied to the opposite side.  Do you mind making this 4-arg?

Reply:
Can if you want, Google has the opposite rule haha hence my default. I'll change it